tests/test_meshes/test_meshes.py

```diff
@@ -175,18 +175,54 @@ def test_ghost_cells(self):
         mesh.add_ghost_meshes()
 
         np.testing.assert_array_equal(
-            mesh["negative electrode_left ghost cell"].edges[1],
-            mesh["negative electrode"].edges[0],
+            mesh["negative electrode_left ghost cell"][0].edges[1],
+            mesh["negative electrode"][0].edges[0],
         )
         np.testing.assert_array_equal(
-            mesh["negative electrode_left ghost cell"].edges[0],
-            -mesh["negative electrode"].edges[1],
+            mesh["negative electrode_left ghost cell"][0].edges[0],
+            -mesh["negative electrode"][0].edges[1],
         )
         np.testing.assert_array_equal(
-            mesh["positive electrode_right ghost cell"].edges[0],
-            mesh["positive electrode"].edges[-1],
+            mesh["positive electrode_right ghost cell"][0].edges[0],
+            mesh["positive electrode"][0].edges[-1],
         )
 
+    def test_multiple_meshes(self):
+        param = pybamm.ParameterValues(
+            base_parameters={
+                "Negative electrode width": 0.1,
+                "Separator width": 0.2,
+                "Positive electrode width": 0.3,
+            }
+        )
+
+        geometry = pybamm.Geometry("1+1D micro")
+        param.process_geometry(geometry)
+
+        # provide mesh properties
+        submesh_pts = {
+            "negative particle": {"r": 5, "x": 10},
+            "positive particle": {"r": 6, "x": 11},
+        }
+        submesh_types = {
+            "negative particle": pybamm.Uniform1DSubMesh,
+            "positive particle": pybamm.Uniform1DSubMesh,
+        }
+
+        mesh = pybamm.Mesh(geometry, submesh_types, submesh_pts)
+
+        # check types
+        self.assertIsInstance(mesh["negative particle"], list)
+        self.assertIsInstance(mesh["positive particle"], list)
+        self.assertEqual(len(mesh["negative particle"]), 10)
+        self.assertEqual(len(mesh["positive particle"]), 11)
+
+        for i in range(1):
```

Comment:
should this be a longer range or not in a for loop?

Reply:
cheers, fixed it